lib/jbuild

```diff
@@ -0,0 +1,13 @@
+(jbuild_version 1)
+
+(library
+ ((name squeeze)
```

Comment:
Don't we need to install the library and give it a `public_name`?

Reply:
I thought nothing was using this library and it was internal to the `squeezed` executable only, we should double check

Reply:
Yes, should be internal only, checked the repos Maccello mentioned, no referrence; actually from original ocamlfind, there is no external lib declared;